packages/arb-token-bridge-ui/src/components/common/Layout.tsx

```diff
@@ -57,6 +60,8 @@ export function Layout(props: LayoutProps) {
         <main>{props.children}</main>
       </div>
 
+      {/* <Toast /> */}
```

Comment:
Any reason we don't enable it in this PR?

Reply:
the original idea is to use toast, but after discussion during standup we think it makes more sense to not show a toast for a user action (rejection), so i removed the toast prompt, but since the toast component was already added and would be useful in the future, i'm keeping the code here